src/Models/CarsModel.php

```diff
@@ -98,6 +148,12 @@ public function insertCar(string $model, string $image_link, int $make_id, strin
         return $query->execute([":inputmodel" => $model, ":inputimage" => $image_link, ":inputmake_id" => $make_id, ":inputbodytype_id" => $bodytype, ":inputyear" => $year, ]);
     }
 
+    public function restoreCar($id)
+    {
+        $query = $this->db->prepare("UPDATE `cars` SET `deleted` = 0 WHERE `id` = :inputid");
+        return $query->execute([':inputid' => $id]);
+    }
+
     public function moveToJunk(int $input_id) : bool
```

Comment:
General comment for this whole file, try and get some more consistency with the param names.

You have:

*   $id
*   $input\_id
*   $inputid
*   $makeid